enhancements/sig-architecture/10-deletepropagationstrategy/README.md

```diff
@@ -0,0 +1,205 @@
+# Introduce DeletePropagation in ManifestWork
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management/website/)
+
+## Summary
+
+Today, when a manifestwork is deleted or a manifest resource in the manifestwork is removed, the work agent
+deletes the related resources on the managedcluster in the `ForegroundDeletion` strategy. It means the manifestwork
+will be kept in deleting status until the related resource owned by the manifestwork is deleted on the managedcluster.
+There is use cases that asks for othere deletion strategies. This proposal is to introduce a property in manifestwork
+that the user can define the orphan deletion propagation for resources in the manifestwork.
+
+## Motivation
+
+### Goals
+
+- Define the DeletePropagation property in manifestwork
+- Indicate the behavior of work agent upon the existence of DeletePropagation property
+
+### Non-Goals
+
+- Deletion strategy other than orphan.
+
+## Proposal
+
+We will introduce a `DeletePropagations` property in the manifestwork API.
+
+### User Stories
+
+#### Story 1
+
+As a manifestwork user, I create a manifeswork with a list of resources. When I delete the manifestwork on the hub,
+I do not want the resources defined in the manifestwork to be garbage collected on the managedcluster.
+
+#### Story 2
+
+As a manifestwork user, I create a manifeswork with a list of resources. When I remove one or several resources from
+the manifestwork, I do not what these resources to be garbage collected on the managedcluster.
+
+
+## Design Details
+
+### Manifestwork API update
+
+We will introduce a `DeletePropagations` property as following:
+
+```go
+// ManifestWorkSpec represents a desired configuration of manifests to be deployed on the managed cluster.
+type ManifestWorkSpec struct {
+	// Workload represents the manifest workload to be deployed on a managed cluster.
+	Workload ManifestsTemplate `json:"workload,omitempty"`
+
+	// DeletePropagationOrphan represents resources to be orphaned when the manifestwork is deleted.
```

Comment:
You want to selectively orphan instead of simply orphaning everything related to it?  Given that a ManifestWork is a unit of related sources, can you elaborate under what conditions you would want to be selective?  As I recall it was primarily geared for cases were a particular resource was shared in some way.

It might make sense to have specifying an orphan policy cover orphaning everything by default and selective orphaning as a secondary option.